tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,5 @@
-<h2 id="title">Welcome to Ember</h2>
+{{!-- The following component displays Ember's default welcome message. --}}
+{{welcome-page}}
```

Comment:
I believe this component is invalid because we don't have the ember-welcome-page addon.